index.html

```diff
@@ -37,24 +30,97 @@
+    </section>
+    <section>
+      <h2>Documentary</h2><img src="DearZacharyTheatricalPoster.jpg" alt="dear zachary poster">
+      <p>Dear Zachary, honestly was both surprising in that I even discovered it but also by how much I was immersed in it. I found this hidden gem, which apparently is very well loved within the documentary fanspace, by scrolling through an article titled “Documentaries with real life plot-twists” and that description is 100% accurate. Dear Zachary was 100% made by Kurt Kunnene whose best friend Dr. Andrew Bagby tragically died by murder before he even knew his wife was pregnant. Said wife, Shirley Turner was the main suspect for the murder and fled to Canada to escape prosecution. Andrew’s parents then have to fight for custody of their grandchild with the individual who murdered their son. Dear Zachary highlights the flawed legal system in Canada and the creation of the Canadian bill C-464. The film was made to bring more awareness towards the consequences of not protecting children within legal battles. I highly suggest watching this film without any research of the case as going into it blind is the reason why it is considered a documentary with a plot-twist. By the end you’ll feel a huge feeling of disappointment and frustration regarding how the Bagby family was treated. Its a heart-wrenching reminder of the lengths humans will go when it comes to someone they love but also the grief and sorrow we experience because of that love.</p>
+    </section>
+    <form action="/server-page-handling-form-data" method="post">
+      <fieldset>
+        <fieldset>
+          <legend>Feedback:</legend>
+          <label for="Topic">Subject/Topic:</label>
+          <input type="text" id="Topic" name="topic" required>
+          
+          <label>Feedback Type:</label>
+          <input type="radio" id="opinion" name="feedback_type" value="opinion" required>
+          <label for="opinion">Opinion</label>
+          
+          <input type="radio" id="recommendation" name="feedback_type" value="recommendation">
+          <label for="recommendation">Recommendation</label>
+          
+          <input type="radio" id="topic_suggestion" name="feedback_type" value="topic_suggestion">
+          <label for="topic_suggestion">Topic Suggestion</label>
+          
+          <label for="feedback">Feedback:</label><br>
+        </fieldset>
+        
+        <textarea id="feedback" name="feedback" rows="4" cols="50" required=""></textarea>
+      </fieldset>
+    </form>
```

Comment:
Your form also needs to be in a section and should have a heading (h2) as well

Reply:
Thank you for the feedback! I'll be sure to look over the tutorial before the midterm! - Amina Farah